### ISSUE TITLE
[Order Creation] Accessibility → Products search → Navigate up button is dictated as "Collapse"

### ISSUE THREAD

**Issue by shiki** [woocommerce/woocommerce-android#5809]
In the products search screen, TalkBack dictates the "Navigate up" button as "Collapse". Weird. 🤷 It should be "Navigate up" like the other screens.

<img src="https://user-images.githubusercontent.com/198826/152878976-7606198e-eace-4834-9b87-07bf22d225c9.png" width="300">

**Comment by nbradbury** [comment]
I believe this is by design. When the search box is showing, tapping the back arrow collapses (closes) it. The same happens elsewhere in the app where we enable searching.
